src/components/Article.jsx

```diff
@@ -1 +1,38 @@
-// export const Article = () => ();
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Article = ({ title, date, text }) => {
+  const today = new Date();
+  const dateISO = today.toISOString().slice(0, 10);
```

Comment:
use your variable date instead of creating new date (just change 'today' with 'date')